.vscode/launch.json

```diff
@@ -41,7 +41,8 @@
                 "${workspaceFolder}/**",
                 "!**/node_modules/**"
             ],
-            "preLaunchTask": "buildDev"
+            "preLaunchTask": "buildDev",
+            "internalConsoleOptions": "openOnSessionStart"
```

Comment:
Not related to the reporting - just an improvement to open the debug output (where test results are printed) when running the integration tests